bin/ch-test

```diff
@@ -978,7 +1002,7 @@ if [[ $phase = one-file ]]; then
 fi
 
 printf "%-21s %s" 'phase:' "$phase"
-if [[ $phase = one-file ]]; then
+if [[ $phase = one-file || $phase = build-unpack ]]; then
```

Comment:
I wonder if there's a reasonable way to make one-file interact with build-unpack. That is, rather than listing images with `-i`, build-unpack builds and unpacks all images *except* if `-f` is given. I think `-f` does have some smarts about which tests indicate an image?